notebooks/workflow_cost_estimator.py

```diff
@@ -0,0 +1,109 @@
+# publish to: "terra-notebook-utils-tests" "test-workflow-cost-estimator"
+import os
+import callysto
+
+# Mock the notebook environment
+os.environ['WORKSPACE_NAME'] = "terra-notebook-utils-tests"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-9169fcd1-92ce-4d60-9d2d-d19fd326ff10"
+os.environ['GOOGLE_PROJECT'] = "firecloud-cgl"
+
+with callysto.Cell("markdown"):
+    """
+    # Workflow Cost Estimator
+    This notebook demonstrates cost estimation for finished or in-progress workflows.
```

Comment:
Suggested addition after this sentence:

"Machine type, start and stop time are pulled from X and compared to current Google Pricing for N1 and N2 machine types. It is important to note that this information only persists for 42 days after a workflow completes."